scalalib/src/mill/scalalib/ScalaModule.scala

```diff
@@ -98,13 +98,19 @@ trait ScalaModule extends mill.Module with TaskModule { outer =>
 
   def platformSuffix = T{ "" }
 
-  def scalaCompilerBridgeSources = T{
+  def scalaCompilerBridgeSources = T {
```

Comment:
We do the same special treatment as here: https://github.com/sbt/zinc/blob/1.x/build.sbt#L391

Reply:
@rockjam seems like a pretty hack way of doing it; is there some way where we won't have to change the code when 2.13.0-final is out? Could we look for the trailing `-Mn` isntead of the leading `2.13.`?

Reply:
sure

Reply:
Fixed it